### PR TITLE
Reduce the acs inactivity timeout to 1 minute

### PR DESCRIPTION
**Pull request by richardpen**
### Summary

This will reduce the acs disconnection period, the maximum disconnection period of acs would be 1 minute before agent try to connect to acs again. This should mitigate the issue #781.
### Implementation details

Change the default heartbeat timeout from 5 minute + random(3minutes) to 1 minute.

### Testing

- [x] Builds on Linux (`make release`)
- [x] Builds on Windows (`go build -out amazon-ecs-agent.exe ./agent`)
- [x] Unit tests on Linux (`make test`) pass
- [x] Unit tests on Windows (`go test -timeout=25s ./agent/...`) pass
- [x] Integration tests on Linux (`make run-integ-tests`) pass
- [x] Integration tests on Windows (`.\scripts\run-integ-tests.ps1`) pass
- [x] Functional tests on Linux (`make run-functional-tests`) pass
- [x] Functional tests on Windows (`.\scripts\run-functional-tests.ps1`) pass
- [x] Run the agent for few hours, check the acs disconnection/connection period.
New tests cover the changes: 

### Description for the changelog

* Enhancement - Reduce the disconnection period of agent to backend.

### Licensing

This contribution is under the terms of the Apache 2.0 License: 
yes

### CODE REVIEW

**Comment by vsiddharth** [comment]
Could you please check the boxes in the summary and also add an explanation as to why the jitter was removed in the commit message body? 

Thanks

File: agent/acs/handler/acs_handler.go [hunk 1]
```diff
@@ -45,8 +45,7 @@ import (
 const (
 	// heartbeatTimeout is the maximum time to wait between heartbeats
 	// without disconnecting
-	heartbeatTimeout = 5 * time.Minute
-	heartbeatJitter  = 3 * time.Minute
+	heartbeatTimeout = 1 * time.Minute
```
Comment (aaithal) [inline agent/acs/handler/acs_handler.go]: Why was the jitter removed?

Comment (jhaynes) [inline agent/acs/handler/acs_handler.go]: @richardpen Can you explain the removal of the jitter here for posterity?

Comment (richardpen) [inline agent/acs/handler/acs_handler.go]: I have added it back in the new commit after understanding why it was there.

Comment (jhaynes) [inline agent/acs/handler/acs_handler.go]: Reading is hard. Thanks!

**Pull request merged**